llama_hub/macrometa_gdn/base.py

```diff
@@ -0,0 +1,84 @@
+"""Macrometa GDN Reader."""
+
+from typing import Any, List
+import requests, json
+
+from llama_index.readers.base import BaseReader
+from llama_index.readers.schema.base import Document
+
+class MagrometaGDNReader(BaseReader):
```

Comment:
Your class name is wrong @dain-macrometa , should be `MacrometaGDNLoader`, probably the CI breaks because of this